assets/sass/alaveteli-org.scss

```diff
@@ -986,3 +986,20 @@ p.ui-dropdown-list-trigger {
 .ui-dropdown-list.active .current-locale:after {
 	border-top-color: #fff;
 }
+
+.fluid-width-video-wrapper {
+    width: 100%;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered padding, padding-top, position, width